static/src/javascripts/lib/defer-to-analytics.js

```diff
@@ -1,10 +1,7 @@
 // @flow
-import config from 'lib/config';
 
 const deferToAnalytics = (afterAnalytics: () => void): void => {
-    try {
-        config.get('modules.tracking.ready').then(afterAnalytics);
-    } catch (e) {} // eslint-disable-line no-empty
+    afterAnalytics();
```

Comment:
Perhaps a daft question, what does this module do, now? Do we still need it? It used to have a side effect that was the cause of the original problem but now it looks pretty benign

Reply:
Yeah, I didn't want to remove it in this PR, it provides a nice decoupling should this solution evolve.